docs/site/Project-layout-reference.md

```diff
@@ -0,0 +1,123 @@
+these sub-directories:
+
+- `src` - TypeScript source code and configuration files.
+- `public` - Client side assets (JavaScript, HTML, and CSS files) for the home
+  page.
+
+{% include tip.html content="By LoopBack naming conventions, artifacts such as models are grouped under their sub-directories. Also, with LoopBack artifact generators, they create class names in CamelCase and file names are in snake-case. For example, if you create a model named `MyUserModel` with the model generator, it creates files `my-user.model.ts` under `src/models`. See [Naming Convention](Command-line-interface.md#naming-convention) for more information.
+" %}
+
+The following table uses files in the
+[`Todo` example](https://github.com/strongloop/loopback-next/tree/master/examples/todo)
+as code references.
+
+<table style="font-size: 90%;">
+  <thead>
+    <tr>
+      <th width="200">File or directory</th>
+      <th>Description</th>
+      <th width="180">How to access in code</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th colspan="3" style="text-align: center; background-color: #bfbfbf;">Top-level application directory</th>
+    </tr>
+    <tr>
+      <td><code>dist/</code> directory</td>
+      <td>Contains transpiled JavaScript code generated by <code>npm run build</code>.</td>
+      <td>npm run build</td>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/strongloop/loopback-next/blob/master/examples/todo/package.json"><code>package.json</code></a></td>
+      <td>
+        Standard npm package specification. Use it to set up package dependencies, among other things. This file must be in the application root directory. See <a href="https://docs.npmjs.com/files/package.json"><code>npm-package.json</code> for information.</a>.
+      </td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td><code>README.md</code></td>
+      <td>Stub file for internal documentation.</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/strongloop/loopback-next/blob/master/examples/todo/tsconfig.json"><code>tsconfig.json</code></a></td>
+      <td>
+        A file specifies the root files and the compiler options required to compile the project. See <a href="https://www.typescriptlang.org/docs/handbook/tsconfig-json.html"><code>tsconfig.json</code> for information.</a>.
+      </td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <th colspan="3" style="text-align: center; background-color: #bfbfbf;">/src directory - Node application files</th>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/strongloop/loopback-next/tree/master/examples/todo/src/controllers"><code>controllers/</code></a> directory</td>
+      <td>A sub-directory for all controller files. See <a href="Controller.html">Controller</a>.</td>
+      <td>&nbsp;</td>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/strongloop/loopback-next/tree/master/examples/todo/src/datasources"><code>datasources/</code></a> directory</td>
```

Comment:
Let's remove `directory` from the description as the trailing `/` already tells so. This comment should apply to other rows too.